### PR TITLE
Fix multiline logs

### PR DESCRIPTION
**Pull request by martinchacon**
Added multiline-log feature

- From v3.2.5 logspout supports multiline-logs [1]

[1] https://github.com/gliderlabs/logspout/tree/v3.2.5#multiline-logging

### CODE REVIEW

File: logspout/modules.go [hunk 1]
```diff
@@ -6,4 +6,5 @@ import (
 	_ "github.com/gliderlabs/logspout/transports/tcp"
 	_ "github.com/gliderlabs/logspout/transports/udp"
 	_ "github.com/looplab/logspout-logstash"
+        _ "github.com/gliderlabs/logspout/adapters/multiline"
```
Comment (matrixik) [inline logspout/modules.go]: File should be formated with gofmt. You could also do it online on https://play.golang.org/

File: logspout/build.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 repository: monasca/logspout
 variants:
-  - tag: 0.0.2
+  - tag: 0.0.3
```
Comment (matrixik) [inline logspout/build.yml]: Please, bump it to `0.1.0`.

File: logspout/README.md [hunk 3]
```diff
@@ -13,6 +13,7 @@ Tags
 Images in this repository are tagged as follows:
 
 * `0.0.1`: standard [SemVer][3].
+* `0.0.3`: added multiline-logs support.
```
Comment (matrixik) [inline logspout/README.md]: I don't think it's needed here, this place is more like description of used tags what are they for.

Comment (martinchacon) [reply logspout/README.md]: Done

File: logspout/build.yml (hunk 2 above)
Comment (martinchacon) [reply logspout/build.yml]: Done

File: logspout/modules.go (hunk 1 above)
Comment (martinchacon) [reply logspout/modules.go]: Done

File: .env [hunk 4]
```diff
@@ -34,6 +34,6 @@ MON_ELASTICSEARCH_CURATOR_VERSION=4.3.1-1.0.0
 MON_LOG_API_VERSION=master-20180627-055222
 MON_KIBANA_VERSION=4.6.3-master
 MON_LOG_AGENT_VERSION=0.0.5
-MON_LOGSPOUT_VERSION=0.0.2
+MON_LOGSPOUT_VERSION=0.1.0
```
Comment (matrixik) [inline .env]: This change needs to be in different PR.
